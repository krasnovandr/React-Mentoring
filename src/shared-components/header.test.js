import React from 'react'
import { render, cleanup } from 'react-testing-library'
import { Route, Link, MemoryRouter } from "react-router-dom";
import Header from "./header"
import renderer from 'react-test-renderer';
// afterEach(cleanup)
describe('Header  Component', () => {
    it('should render error component', () => {
        const component = renderer.create(
            <Header />
        );

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();

    })
})