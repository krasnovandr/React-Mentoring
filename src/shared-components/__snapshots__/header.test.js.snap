// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header  Component should render error component 1`] = `
<div
  className="Header-root-1"
/>
`;
